src/components/events/adminEvent/AdminEvents.jsx

```diff
@@ -1,9 +1,11 @@
 import { React, useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
 import { Input, Button, Radio, Row, Col, Card, Typography, ConfigProvider } from 'antd';
-import { SearchOutlined } from '@ant-design/icons';
+import { FilterOutlined, PlusOutlined, SearchOutlined } from '@ant-design/icons';
 import axios from 'axios';
 import EventCard from '../EventCard';
+import EventList from '../EventList';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module '../EventList'.